src/virt-install

```diff
@@ -271,6 +292,7 @@ try:
     if args.variant == 'metal-uefi':
         vinstall_args.append('--boot=uefi')
     run_sync_verbose(vinstall_args)
+    print(f"Served {n_requests}")
```

Comment:
Maybe `Served {n_requests} objects` ?

Reply:
It's not just objects though, we e.g. serve the `/config` request this way.  I reworded it.

Reply:
Not to belabour this too much, but it still seems like a noun is missing here, no? Did you mean `Served {n_requests} requests via ...` ?

Anyway, LGTM either way, feel free to merge!